index.html

```diff
@@ -71,7 +71,9 @@
 	<li class="delete separator"><a href="#delete"></a></li>
 </ul>
 
-<script type="text/javascript" src="scripts/jquery-1.8.3.min.js"></script>
+<script type="text/javascript" src="vendor/compoents/jquery/jquery.min.js"></script>
```

Comment:
@simogeo 
should i update only the jquery-version and not load from the composer?